### ISSUE TITLE
[BITV] 9.4.1.2/2.7 - "Weather" Menu - The "Favorites" button is an accordion button that should be implemented as a button with an aria-expanded attribute, without using role="menuitem". (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37100]
See https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/

- [ ] - [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v2-n7

</details>
